packages/core/saved-objects/core-saved-objects-migration-server-internal/src/initial_state.ts

```diff
@@ -151,7 +151,9 @@ export const createInitialState = ({
     retryDelay: 0,
     retryAttempts: migrationsConfig.retryAttempts,
     batchSize: migrationsConfig.batchSize,
+    defaultBatchSize: migrationsConfig.batchSize,
```

Comment:
NIT I wonder if `defaultBatchSize` won't be a bit confusing (the real default was 1000).
WDTY about keeping `maxBatchSize` (which has a default of 1000) and defining `currentBatchSize` instead?

Reply:
**UPDATE** or perhaps using `batchSize` for the current one, as you do ATM, and doing `maxBatchSize` instead of `defaultBatchSize`.